tests/functional/disaster-recovery/metro-dr/test_replace_cluster.py

```diff
@@ -0,0 +1,210 @@
+import logging
+import pytest
+import time
+
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.framework import config
+from ocs_ci.ocs.acm.acm import AcmAddClusters
+from ocs_ci.ocs import constants
+
+from ocs_ci.framework.pytest_customization.marks import tier1, turquoise_squad
+from ocs_ci.helpers.dr_helpers import get_active_acm_index
+from ocs_ci.ocs.node import get_node_objs
+from ocs_ci.ocs.dr.dr_workload import validate_data_integrity
+from ocs_ci.helpers.dr_helpers_ui import (
+    check_cluster_status_on_acm_console,
+    failover_relocate_ui,
+)
+from ocs_ci.helpers.dr_helpers import (
+    replace_cluster,
+    enable_fence,
+    enable_unfence,
+    get_fence_state,
+    failover,
+    relocate,
+    set_current_primary_cluster_context,
+    get_current_primary_cluster_name,
+    get_current_secondary_cluster_name,
+    wait_for_all_resources_creation,
+    gracefully_reboot_ocp_nodes,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@turquoise_squad
+class TestReplaceCluster:
+    """
+    Test for Recovery of replacement cluster
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request, cnv_dr_workload):
+        """
+        Teardown function: If fenced, un-fence the cluster and reboot nodes
```

Comment:
how do you plan to address the test failure at various phases/steps that need clean up? For example, the test fails at import cluster phase..

Reply:
this is a non-blocking comment and can be addressed in future PRs. Please have a github issue created